server/src/test/java/org/elasticsearch/transport/RemoteClusterConnectionTests.java

```diff
@@ -142,6 +145,18 @@ public static MockTransportService startTransport(
                     }
                 }
             );
+            newService.registerRequestHandler(
```

Comment:
do we need to register both old and new action ?

Reply:
this question is unanswered, maybe the answer is yes :)

Reply:
Good catch. I've removed the old action in https://github.com/elastic/elasticsearch/pull/95894/commits/b3803e12c7811504c656046ba03923ec7324b7b9.